dspace/config/modules/controlpanel.cfg

```diff
@@ -0,0 +1,18 @@
+#---------------------------------------------------------------#
+#------------ADMIN CONTROL PANEL CONFIGURATIONS-----------------#
+#---------------------------------------------------------------#
+
+### Control Panel Tabs
+controlpanel.tabs = Java Information, \
```

Comment:
The panel headers cannot be internationalized, best to use a generic identifier & generate the xmlui message using that as opposed to placing them in config.